docs/machine-learning/basic-concepts-model-training-in-mldotnet.MD

```diff
@@ -0,0 +1,169 @@
+---
```

Comment:
I found several instances of 1st person text in this article. I strong suggest running Acrolinx on it and switching those to 2nd person.

Reply:
+1